source/download/index.html.haml

```diff
@@ -157,11 +157,13 @@ authors: bproffitt, dneary, knesenko, mburns, sandrobonazzola, theron
   |Windows XP Service Pack 3 and newer|32-bit||Yes|
   |Windows 7|32-bit, 64-bit||Yes|
   |Windows 8|32-bit, 64-bit||No|
+  |Windows 10|32-bit, 64-bit||No|
```

Comment:
@didib @lveyde is ovirt-guest-tools providing support for spice on windows >= 8?